course/page/text.py

```diff
@@ -41,6 +41,8 @@
 import re
 import sys
 
+CA_PATTERN = string_concat(_("A correct answer is"), ": '%s'.")  # noqa
```

Comment:
Since it's now global (so that context is no longer immediately obvious, I'd prefer `CORRECT_ANSWER_PATTERN`.

Reply:
Done.